adapter/internal/api/apis_impl.go

```diff
@@ -151,6 +174,31 @@ func extractAPIProject(payload []byte) (apiProject ProjectAPI, err error) {
 	return apiProject, nil
 }
 
+func verifyMandatoryFields(apiJSON config.APIJsonData) error {
+	var errMsg string = ""
```

Comment:
apiJson.Data can be null?

Reply:
no can't

Reply:
It can be null? 
try deploying apictl project after deleting the `data` segment

Reply:
data segment could be removed from ctl project?